tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarCluster.java

```diff
@@ -72,10 +83,13 @@ public static PulsarCluster forSpec(PulsarClusterSpec spec) {
     private final ProxyContainer proxyContainer;
 
     private PulsarCluster(PulsarClusterSpec spec) {
+
         this.spec = spec;
         this.clusterName = spec.clusterName();
         this.network = Network.newNetwork();
+
         this.zkContainer = new ZKContainer(clusterName)
+            .withLogConsumer(new Slf4jLogConsumer(log).withPrefix(ZKContainer.NAME))
```

Comment:
you already added Sfl4jLogConsumer to PulsarContainer, I don't think you need to configure it again here.